OpenSourcePolitics/decidim-cd44
# fix: User Answers not exporting from surveys

#### :tophat: Description
*Please describe your pull request.*

#### :pushpin: Related Issues
*Link your PR to an issue*
- [Error on Decidim::Forms::ExportQuestionnaireAnswersJob](https://opensourcepolitics.odoo.com/web?db=opensourcepolitics&signup_email=guillaume%40opensourcepolitics.eu&token=ixqNZ9S5lGRCj9ADpsQq#id=3105&cids=1&menu_id=325&action=471&model=project.task&view_type=form)

#### Testing
*Describe the best way to test or validate your PR.*

* Initialize your app 

### TO SETUP YOUR APP
- Please use DOCKER you absolutely need it
- If you already had setup a docker instance of decidim-tou in the past please run `make teardown`
- Run the command `make run` to setup your app
- Once it's setup please make sure you access to your platform using the url `https://localhost:3000`

### TEST THE FEATURE
- Log in as a user (or admin)
- Access to BackOffice
- Go to a Participatory Process
- Go to the Survey component settings
- Open it to the answers
- Access front office using your account or an other
- Answer it and submit it 
- Go back to the BackOffice
- Access the answers (using edit button on the survey component)
- Export an answer or every answers
- Check on letter_opener if you have the export
- Check on Sidekiq if you have the error on an other one and please share it to us

#### Tasks
- [x] Fix version of wicked_pdf to 2.6.3
- [x] Enable import of wicked_pdf on `config/application.rb`
- [x] Fix the CI

### :camera: Screenshots
*Please add screenshots of the changes you're proposing if related to the UI*
